Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -183,8 +165,8 @@
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Workspace" />
     <Reference Include="Microsoft.VisualStudio.Workspace.VSIntegration.Contracts" />
-    <Reference Include="Newtonsoft.Json, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
-      <HintPath>$(PackagesPath)\Microsoft.VSSDK.BuildTools\tools\vssdk\bin\lib\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
did this fix the Newtonsoft.Json warning?

Reply:
I wish. This was necessary to get it to build with D16